app/services/discovery_report.rb

```diff
@@ -76,9 +77,12 @@ def file_manifest
   # By using jbuilder on an enumerator, we reduce memory footprint (vs. to_a)
   # @return [Jbuilder] (caller needs obj.to_builder.target! for the JSON string)
   def to_builder
-    Jbuilder.new do |json|
+    json_report = Jbuilder.new do |json|
       json.rows { json.array!(each_row) }
       json.summary summary
     end
+    @objects_had_errors = (@summary[:objects_with_error] > 0) # indicate if any objects generated errors
```

Comment:
could this be `summary` instead of `@summary`?   I lose track of when we need the `@` ... but maybe just when assigning?